### ISSUE TITLE
Не работает Bot LongPoll

### ISSUE THREAD

**Issue by xxsxchi** [python273/vk_api#178]
В группе все включено, токен группы валиден, лонгпул включен и работает на версии 5.84. При запуске скрипта из примера в консоль ничего не выводится.

**Comment by Helow19274** [comment]
Вопрос, конечно, глупый. А события точно происходят? А то если в группе ничего не происходит, ничего и не выведется

**Comment by xxsxchi** [comment]
Точно происходят. Входил и выходил из сообщества с другого аккаунта, ничего не пишет. Так же и с сообщениями. Кидал другу, у него тоже самое.

**Comment by VasyaDolmat1917** [comment]
да, кстати. Писал в лс группы. Отписывался, подписывался, но ничего в консоль не выводит

**Comment by Helow19274** [comment]
Код как [тут](https://github.com/python273/vk_api/blob/master/examples/bot_longpoll.py)?

**Comment by xxsxchi** [comment]
Да, как там

**Comment by VasyaDolmat1917** [comment]
да, такой

**Comment by VasyaDolmat1917** [comment]
Senks забыл
